source/content/drupal-9.md

```diff
@@ -0,0 +1,91 @@
+---
+title: Drupal 9 Early Access
+description: Test the latest Drupal on the Pantheon Website Management Platform.
+categories: [get-started]
+tags: [site]
+earlyaccess: true
+contributors: [populist, edwardangert]
+reviewed: "2020-05-21"
+---
+
+Drupal 9 is, according to [drupal.org](https://www.drupal.org/docs/understanding-drupal/drupal-9-resources), "a cleaned-up version of Drupal 8... \[with] deprecated code removed and third-party dependencies updated."
+
+Drupal 9 updates Drupal’s underlying dependencies like [Symfony 4.4](https://symfony.com/releases/4.4) and [Twig 2](https://twig.symfony.com/doc/2.x/index.html), removes several deprecated API functions in favor of better options, and allows everyone running Drupal 8.8+ an easy upgrade path to Drupal 9 & beyond.
+
+<Alert title="Warning" type="danger">
```

Comment:
Does the heading need to say "warning" ? Can we move the below in bold into the heading?

"Early Access Only - Not for Production Use"